### PR TITLE
Updates for laravel 6

### PR DESCRIPTION
**Pull request by SudoGetBeer**
Hello,

@hisorange hope this helps you a bit.
Just changed everthing to make this work.
The change in ParserTest was because of an Warning from PhpUnit

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,17 +12,17 @@
         }
     ],
     "require": {
-        "php": ">=5.6.0",
-        "laravel/framework": "~5.0",
+        "php": ">=7.2",
+        "laravel/framework": "^6.0",
```
Comment (matthewnessworthy) [inline composer.json]:
```suggestion
        "laravel/framework": "^5.0|^6.0",
```

File: composer.json [hunk 2]
```diff
@@ -12,17 +12,17 @@
         }
     ],
     "require": {
-        "php": ">=5.6.0",
-        "laravel/framework": "~5.0",
+        "php": ">=7.2",
```
Comment (matthewnessworthy) [inline composer.json]: i think you should revert this change, otherwise laravel this package will break for all users not using Laravel 6
```suggestion
        "php": ">=5.6.0",
```

File: composer.json [hunk 3]
```diff
@@ -12,17 +12,17 @@
...
         "ua-parser/uap-php": "~3.8",
         "league/pipeline": "~0.1",
         "mobiledetect/mobiledetectlib": "~2.0",
         "jaybizzle/crawler-detect": "~1.0",
         "piwik/device-detector": "~3.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "~5.0 || ~6.0 || ~7.0",
-        "orchestra/testbench": "~3.0",
+        "phpunit/phpunit": "~8.0",
```
Comment (matthewnessworthy) [inline composer.json]: why are these dev dependency changes required?

Comment (SudoGetBeer) [reply composer.json]: Because Laravel needs testbench 3.9 (or 4) and testbench needs phpunit 8. 
Will change it to:
`
"phpunit/phpunit": "~5.0 || ~6.0 || ~7.0 || ~8.0",
        "orchestra/testbench": "~3.0",
`

**Comment by SudoGetBeer** [comment]
Thanks @matthewnessworthy. Applied your suggestions.

**Comment by stevenwoodson** [comment]
For sake of completeness, it may be helpful to also update the `.travis.yml` with Laravel 6 tests and the `readme` to update version support. I hope this gets merged soon!

**Comment by dmason30** [comment]
@hisorange Please can you look at this. Am using a fork for the time being but prefer to avoid that. Much appreciated. Thanks

**Comment by markdieselcore** [comment]
Please merge.

**Comment by hisorange** [comment]
Hey there, just merging this. I will have to do some more change.

Sincerely thank You for Your work!
